chart/kubeapps/values.yaml

```diff
@@ -1818,9 +1818,12 @@ redis:
       ## allkeys-lru: evict keys by trying to remove the less recently used (LRU) keys first, in order 
       ## to make space for the new data added
       - "--maxmemory-policy allkeys-lru"
+      ## ref https://stackoverflow.com/questions/22815364/flushall-and-flushdb-commands-on-redis-return-unk-command
+      ## Redis official Helm chart by default disables FLUSHDB and FLUSHALL commands
+    disableCommands: []
```

Comment:
Did you mean to disable those? Or just indicate here that it can be used? Unless we're populating/using it here explicitly, not sure why it's added to our chart.

Also note the comment indentation will need to line up with the field for the readme generator, I think (not 100%). (for the replica below too).

Reply:
Yes, this is intentional. Bitnami disables FLUSHDB command by default (WTF?) and I need it, so this is what its here for. I will fix the indent